src/cow-react/common/pure/CurrencyInputPanel/CurrencyPreview.tsx

```diff
@@ -35,10 +36,10 @@ export function CurrencyPreview(props: CurrencyPreviewProps) {
           </div>
           <div>
             <styledEl.NumericalInput
-              title={rawAmount?.toExact() + ' ' + currency?.symbol}
+              title={FractionUtils.fractionLikeToExactString(rawAmount)}
```

Comment:
What about the symbol?

Reply:
Thanks! Replaced by `<TokenAmount>` component